howto/instance/access-instance.md

```diff
@@ -1,9 +1,66 @@
 (howto-access-instance)=
 # How to access an instance
 
-In some cases, it might be necessary to access an individual instance for debugging reasons.
+In some cases, it might be necessary to access an individual instance for debugging or development purposes.
 
-You can do this on the command line with the `amc` command, or you can use [scrcpy](https://github.com/Genymobile/scrcpy) for graphical access.
+You can:
+- Run `anbox-connect` for secure ADB connections.
+- Access an instance using the `amc` command for shell access.
+- Use [scrcpy](https://github.com/Genymobile/scrcpy) for graphical access to Android instance.
+
+## Access Android instance using anbox-connect
+
+Secure ADB connection is a feature introduced in Anbox Cloud 1.23.0 that allows users to create a secure ADB connection through the `anbox-connect` command without exposing the [ADB service](https://anbox-cloud.io/docs/howto/instance/expose-services).
+
+### Install anbox-connect snap
+
+First, install the `anbox-connect` snap from the snap store:
+
+    snap install anbox-connect
+
+### Generate a presigned connection URL
+
+Generate a presigned connection URL in the following ways:
+
+- **From the Dashboard**:
+  Navigate to the `Instances` view from the left panel, locate a running instance, click the ![Connect ADB|16x16](https://assets.ubuntu.com/v1/tbd_connect-adb.png) icon button, then click `Authorize` in the popup dialog and copy the command provided.
+
+- **From the Command Line**:
+  * For a full Anbox Cloud deployment:
+
+        anbox-stream-gateway session share <session_id>
+
+  * For the Anbox Cloud Appliance:
+
+        anbox-cloud-appliance.gateway session share <session_id>
+
+  This command will return a presigned URL that you can use to connect to the remote Android instance.
+
+### Connect to the Instance
+
+Open a terminal and use the `anbox-connect` command to establish a secure ADB connection to the remote Anbox instance:
```

Comment:
So here the connection is to the Anbox instance? Earlier, we were talking about "Accessing the Android instance" This is a bit confusing for first time users.

Reply:
We discussed this, this will taken care with the separation of content related to Anbox instance vs Android instance.

Reply:
Yes, as discussed, we mixed the concept of `Access Anbox instance` and `Access Android instance` up in this page, 
We will move `Access Android instance using anbox-connect` alongside `Access Android instance using scrcpy` into a separate page under the `Work with Android`

Reply:
Technically, the connection is to adbd(adb daemon running inside of Android container) through the anbox session. I think it'd be better to say `to the remote Android sinace`